analysis/absrisks_tidy_for_viz.R

```diff
@@ -0,0 +1,152 @@
+# manuscript
+irs_crude <- 
+  map(.x = input_files_irs_crude,
+      .f = ~ read_csv(.x,
+                      col_types = cols_only(subgroup = col_character(),
+                                            level = col_character(),
+                                            rate_redacted = col_double(),
+                                            lower_redacted = col_double(),
+                                            upper_redacted = col_double())) %>%
+              filter(subgroup == "agegroup") %>%
+              rename(ir = rate_redacted,
+                     lower = lower_redacted,
+                     upper = upper_redacted))
+# combine agegroup from crude file and rest
+estimates <-
+  map2(.x = irs_crude,
+       .y = irs_std,
+       .f = ~ bind_rows(.x, .y))
+names(estimates) <- c("wave1", "wave2", "wave3")
+# vax coverage data
+input_files_coverage <-
+  Sys.glob(here("output", "tables", "wave*_vax_coverage.csv"))
+coverage <- 
+  map(.x = input_files_coverage,
+      .f = ~ read_csv(.x,
+                      col_types = cols_only(subgroup = col_character(),
+                                            level = col_character(),
+                                            cov_2 = col_double())) %>%
+        filter(subgroup != "region"))
+names(coverage) <- c("wave1", "wave2", "wave3")
+
+# Combine the estimates and vax coverage ---
+est_cov_combined <- 
+  map2(.x = estimates,
+       .y = coverage,
+       .f = ~ .y %>% 
+         full_join(.x, by = c("subgroup", "level")))
+
+# Process the combined estimates and vax coverage ---
+## uses function 'process_est_cov_combined' defined above
+est_cov_processed <-
+  map(.x = est_cov_combined,
+      .f = ~ process_est_cov_combined(.x,
+                                      subgroups_and_plot_groups))
+## Make one wide table from list of processed tables
+table_est_cov <-
+  est_cov_processed$wave1 %>%
+  left_join(est_cov_processed$wave2,
+            by = c("subgroup", "level", "plot_category", "plot_group"),
+            suffix = c(".1", ".2")) %>%
+  left_join(est_cov_processed$wave3,
+            by = c("subgroup", "level", "plot_category", "plot_group"))
+## add suffix '.3' to indicate wave 3 results
+colnames(table_est_cov)[c(13, 14, 15, 16)] <- 
+  paste0(colnames(table_est_cov)[c(13, 14, 15, 16)], ".3")
+## select columns needed + calculate ratio of IR
+table_est_cov <- 
+  table_est_cov %>%
+  select(-c(cov_2.1, cov_2.2)) %>%
+  rename(Characteristic = subgroup,
+         Category = level,
+         Plot_category = plot_category,
+         Plot_group = plot_group,
+         Coverage = cov_2.3) %>%
+  mutate(IR_ratio.2 = IR.2 / IR.1,
+         IR_ratio.3 = IR.3 / IR.1)
+
+# Save output --
+## saved as '/output/tables/wave*_vax_coverage.csv
+output_dir <- here("output", "tables")
+ifelse(!dir.exists(output_dir), dir.create(output_dir), FALSE)
+write_csv(table_est_cov,
+          path = paste0(output_dir,
+                        "/absrisks_for_viz_tidied.csv"))
```

Comment:
```suggestion
          path = fs::path(output_dir, "absrisks_for_viz_tidied.csv"))
```

`fs::path` will take care of folder delimiters across different systems for you, so it's a bit less error prone